### ISSUE TITLE
Duplicate semantic error during EnC

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#19234]
Step into Main method in the following console app:

```C#
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine(2);
    }
}

class C
{
    void F(int a, int b = 1)
    {
    }

    void G()
    {
        F(100);
    }
}
```

While debugging add an overload of F to class C like so:

```C#
    void F(int a, double b = 1)
    {
    }
```

An error is reported in Error List as expected.

Change the call ```F(100)``` in method G to ```F(1)``` and Continue.

A dialog blocking the Continue action is shown as expected.
The error list now contains a duplicate entry for the error.

![image](https://cloud.githubusercontent.com/assets/41759/25686482/857272f4-3023-11e7-8f3a-f369d3456367.png)

It should only have one entry.
